deps/build_hsl_ma57.jl

```diff
@@ -1,10 +1,10 @@
-open(hsl_ma57_archive) do f
-  if bytes2hex(sha256(f)) != hsl_ma57_sha256
-    error("SHA256 of HSL MA57 doesn't match")
-  end
+if hsl_ma97_version.ext == ".tar.gz"
+    run(`tar -zxf $hsl_ma57_archive -C $builddir`)
+elseif hsl_ma97_version.ext == ".zip"
+    run(`unzip $hsl_ma57_archive -d $builddir`)
```

Comment:
Many thanks for your contribution! I haven't had a chance to test yet, but I suppose the above should say `ma57` instead of `ma97`?!

Reply:
Nice catch. This has now been fixed.